lib/uiautomator2.js

```diff
@@ -143,14 +143,14 @@ class UiAutomator2Server {
   async startSessionUsingAdbKit (deviceUDID) {
     logger.info(`Running command: 'adb -s ${deviceUDID} shell am instrument -w ${INSTRUMENTATION_TARGET}'`);
     this.client.shell(deviceUDID, `am instrument -w ${INSTRUMENTATION_TARGET}`)
-      .then(adbkit.util.readAll)
-      .then(function (output) {
+      .then(adbkit.util.readAll) // eslint-disable-line promise/prefer-await-to-then
```

Comment:
Ah. Turns out I did have an idea and forgot. 

Anyway, we're not actually waiting here at all. Just setting up the logging from the adb command, which runs in the background the whole time. We _could_ separate out the `adbkit.util.readAll` command but it would remove the error logging.